src/components/operations/operation-details/ko/runtime/operation-console.html

```diff
@@ -140,8 +140,8 @@ <h3>Parameters</h3>
                     </div>
                 </div>
             </div>
-            <div class="col-2">
-                <a href="#" role="button" data-bind="click: $component.removeQueryParameter">Remove</a>
+            <div class="col-1">
+                <a href="#" role="button" data-bind="click: $component.removeQueryParameter" class="remove-button remove-inline" aria-label="Remove parameter"><i class="icon-emb icon-emb-trash"></i></a>
```

Comment:
`class="remove-button remove-inline"` - I have a suspicion that this could be `no-border` class instead of these two new classes. Please take a look at [authorizaion.html](https://github.com/Azure/api-management-developer-portal/blob/master/src/components/operations/operation-details/ko/runtime/authorization.html#L3) and how this component looks like in the console.

There is a small with your current approach: This way we introduce pattern of adding a new CSS class for each new action (in this case "remove-button"). As mentioned above, we may need just a style variation like "no-border" or "btn-no-border" (or something like this) that would apply to all similarly looking buttons (Please take a look at the Bootstrap approach for buttons [here](https://getbootstrap.com/docs/5.1/components/buttons/)).